pkg/engine/engine.go

```diff
@@ -719,7 +724,11 @@ func getContainerAddonsString(properties *api.Properties, sourcePath string) str
 		if setting.isEnabled {
 			var input string
 			if setting.rawScript != "" {
-				input = setting.rawScript
+				var err error
+				input, err = getStringFromBase64(setting.rawScript)
```

Comment:
How are we dealing with gzip'd input?

Reply:
We are not. This PR is implemented as per what we've mentioned in the documentation. It's stated that the user must provide the B64 encoded string of the addon YAML

Reply:
The documentation states gzip+base64, and in that order:

https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md#addons

(also the title of this PR suggests gzip)

Reply:
Oh yes. There seems to be a discrepancy.